packages/sdk/src/storage-manager/getStorageManager.ts

```diff
@@ -17,14 +18,21 @@ import { StorageManagerAS as SMDyn } from './StorageManagerAS';
 //#endif
 
 export const getStorageManager = (
-  // platformManager: PlatformManager,
   options: StorageManagerProps,
 ): StorageManager => {
-  // TODO uncomment and test to use similar dynamic imports for each platforms and drop support for JSCC
-  // Currently might have an issue with NextJS and server side rendering
-  // if (platformManager.isNotBrowser()) {
-  //   const { StorageManagerNode } = await import('./StorageManagerNode');
-  //   return new StorageManagerNode(options);
-  // }
-  return new SMDyn(options);
+  if (PlatformManager.isBrowser()) {
+    return new SMDyn(options);
```

Comment:
Should this be dynamically imported? Will nextjs complain that browser APIs are still being imported?